playbooks/openstack-enroll_caasp_workers.yml

```diff
@@ -46,6 +46,19 @@
         dest: "{{ upstream_repos_clone_folder }}/ccp/"
 
 
+    # In SLE15SP1 the default bundler is 2.x while the velum Gemfile.lock has been bundled with 1.15.0 so we need to
```

Comment:
Could you please add a comment that this change is only needed for CaaSP3 and can be removed once we switch to CaaSP4?

Reply:
Good point!